### ISSUE TITLE
whitelist, blacklist alternatives

### ISSUE THREAD

**Issue by MarianneAW** [MicrosoftDocs/microsoft-style-guide#164]
[ Wondering whether MS Style would weigh in on allow list or allowlist, deny list or denylist as acceptable alternatives to whitelist and blacklist., including whether these terms can be used as nouns and verbs or only as nouns.]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a09d7650-47d1-1315-9899-0c8c2ede75cb
* Version Independent ID: 7e1c7ede-8eed-c87d-8548-f269a99c3f2a
* Content: [whitelist - Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/whitelist)
* Content Source: [styleguide/a-z-word-list-term-collections/w/whitelist.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/master/styleguide/a-z-word-list-term-collections/w/whitelist.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**

**Comment by GossiTheDog** [comment]
Yes, this should say allow list and deny list in line with code base.

**Comment by S-LRaillard** [comment]
Hi @GossiTheDog @MarianneAW , just a heads up that in case you didn't catch it, in 2021 Microsoft moved away from "whitelist" and "blacklist" to suggest "allow list" and "block list' or other alternatives as appropriate. Please consult the revelant style guide topics for more info.
